### ISSUE TITLE
Where do I put my Thermostat URL?   

### ISSUE THREAD

**Issue by kevinsickles** [bbrookfield/alexa-radiothermostat#2]
a NOOB,   when I replace the THERMOSTAT_URL   with my thermostat ip address   I get

/home/pi/node-alexa-server/app_modules/alexa-radiothermostat/index.js:8
  request.post(process.env.192.168.15.75 + '/tstat', {json: {t_cool: parseFloa
                          ^^^^
SyntaxError: Unexpected number

**Comment by bbrookfield** [comment]
Remove the process.env. before your IP Address. The process.env was to  get the ip address from a private var on heroku, you do not need it. replace with http://192.168.15.75

**Comment by bbrookfield** [comment]
change the line from
request.post(process.env.192.168.15.75 + '/tstat', {json: {t_cool: parseFloa
to
request.post('http://192.168.15.75/tstat', {json: {t_cool: parseFloa

**Comment by kevinsickles** [comment]
Thank you!    Next problem is the endpoint is not responding.  I assume I have to set up SSL on the node.js express server?

**Comment by bbrookfield** [comment]
Yes, Amazon requires ssl (HTTPS) for any skill endpoint. I believe you also cannot use a self signing certificate. This is why I suggest using Heroku cloud service.

**Comment by jbonzey** [comment]
I have a veraedge connected to Amazon echo through amazon aws.  I was hoping to use your radiothermostat app in another lambda within my existing account.  I know this may be an insane question but what I'm trying to do is just use your app code with aws.  As you can tell i'm  new at this.  Thanks
